app/services/address_exporter.rb

```diff
@@ -35,11 +34,14 @@ def self.se_mailing_csv_str(address)
 
 
 
-  # if country == 'Sverige' return what the Swedish Postal service wants to see for a mailing address
+  # if country == 'Sverige' || 'SE' || 'Sveriges' standardize to "Sverige"
   def self.country_postal(address)
 
     if address
-      address.country == SVERIGE_COUNTRY ? SVERIGE_POSTAL_STR : address.country
+      if SVERIGE_VARIATIONS.include?(address.country)
+        address.country = SVERIGE_COUNTRY
+      end
```

Comment:
I wouldn't expect a export function to change the data. Is this intentional?

Reply:
I don't know. I don't know if the data has a country. 
Why not try and remove the code regarding country, and see what is there then? 
Maybe that actually solves the issue?

Reply:
@thesuss I am not sure I understand what you're saying. My remark was about the export function having the side effect of changing the original data (the county field). it can be easily solved by introducing a local variable. I just wasn't sure if this was intentional or not.

Reply:
I think it is, but I'm not sure it has to be.